doc/classes/PrimitiveMesh.xml

```diff
@@ -13,7 +13,12 @@
 			<return type="Array">
 			</return>
 			<description>
-				Returns mesh arrays used to constitute surface of [Mesh]. Mesh arrays can be used with [ArrayMesh] to create new surfaces.
+				Returns mesh arrays used to constitute surface of [Mesh]. The result can be passed to [method ArrayMesh.add_surface_from_arrays] to create a new surface. For example:
+				[codeblock]
+				var c := CylinderMesh.new()
```

Comment:
Not critical here, but we generally don't use type hints in code samples.

Reply:
oops, will keep that in mind 👍